src/blocks/button.css

```diff
@@ -0,0 +1,21 @@
+.card__button {
```

Comment:
file name is `button`, but you're describing elements of `card` block, [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 10th item - you should only create new file for different blocks and keep there it's elements too